### PR TITLE
Add SIG-Community

### PR DESCRIPTION
**Pull request by evan2645**
We are beginning a new bi-weekly call for managing community-related
efforts, including events, conference planning, growth tracking and
measurement, and social media.

Umair Khan will be the first SIG lead as the current SPIRE community
chair.

Signed-off-by: Evan Gilman <egilman@vmware.com>

### CODE REVIEW

**Comment by umairmkhan** [comment]
Thanks, @evan2645  Looking forward to hosting these.

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Most community activity is organized into Special Interest Groups (SIGs), time-b
 
 | Name | Lead | Group | Slack Channel | Meetings |
 |:------:|:-------:|:-------:|:---------------:|:----------:|
+| [SIG-Community](/community/sig-community/README.md) | [Umair Khan](https://github.com/umairmkhan) (HPE) | [Here](https://groups.google.com/a/spiffe.io/g/sig-community) | [Here](https://spiffe.slack.com/messages/community) | [Notes](https://docs.google.com/document/d/1tb3lxubwr8IKRd6Smnl83ur14xkOQdjwQqla9OHjwZo) |
```
Comment (amartinezfayo) [inline README.md:64]:
```suggestion
| [SIG-Community](/community/community/README.md) | [Umair Khan](https://github.com/umairmkhan) (HPE) | [Here](https://groups.google.com/a/spiffe.io/g/sig-community) | [Here](https://spiffe.slack.com/messages/community) | [Notes](https://docs.google.com/document/d/1tb3lxubwr8IKRd6Smnl83ur14xkOQdjwQqla9OHjwZo) |
```

Comment (evan2645) [reply README.md:64]: Good catch @amartinezfayo 

I opted to rename the directory instead

**Comment by azdagron** [comment]
This is looking pretty great! I'm happy with it.

**Review by azdagron — APPROVED** [review]

**Review by anvega — APPROVED** [review]
👍

**Pull request merged**